### PR TITLE
Unsupported country modal: extend supported countries list

### LINKED ISSUE #2452: Unsupported countries modal: mark new 8 countries as supported  (AT, BE, NL, PL, PT, CH, HK, SG)
When starting onboarding, we check if countries are supported. The list of supported countries needs to be extended.

### PR DESCRIPTION
**Pull request by kalessil**
Fixes #2452 

#### Changes proposed in this Pull Request

- Adds new countries to the list of supported for the modal

#### Testing instructions

- Change store country to one of the new countries
- Test onboarding - when starting onboarding, modal should not appear

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by kalessil** [comment]
Marking as blocked until 896-gh-Automattic/woocommerce-payments-server and 895-gh-Automattic/woocommerce-payments-server are ready.

**Review by naman03malhotra — APPROVED** [review]
Tested, works! LGTM, :shipit:

**Comment by naman03malhotra** [comment]
Getting this when redirected to stripe, I am guessing I have to switch to the appropriate backend branch for this to work?
image

**Comment by kalessil** [comment]
That looks like the country is not enabled for the platform account. That's ok in this testing scenario.

File: includes/class-wc-payments-utils.php [hunk 1]
```diff
@@ -177,15 +177,24 @@ public static function zero_decimal_currencies(): array {
...
 			'CA' => __( 'Canada', 'woocommerce-payments' ),
+			'CH' => __( 'Switzerland', 'woocommerce-payments' ),
 			'DE' => __( 'Germany', 'woocommerce-payments' ),
 			'ES' => __( 'Spain', 'woocommerce-payments' ),
 			'FR' => __( 'France', 'woocommerce-payments' ),
 			'GB' => __( 'United Kingdom (UK)', 'woocommerce-payments' ),
+			'HK' => __( 'Hong Kong', 'woocommerce-payments' ),
 			'IE' => __( 'Ireland', 'woocommerce-payments' ),
 			'IT' => __( 'Italy', 'woocommerce-payments' ),
+			'MY' => __( 'Malaysia', 'woocommerce-payments' ),
```
Comment (thenbrent) [inline includes/class-wc-payments-utils.php:192]: Malaysia should not be included just yet. More details: paJDYF-1nZ-p2#comment-7117 and pbUcTB-k2-p2#comment-1493

Comment (kalessil) [reply includes/class-wc-payments-utils.php]: Thanks for the ping! Reverted in def8b81cd5ccd22547a74455f34e5b9f037ae6e6

**Comment by kalessil** [comment]
The PRs referenced previously are finished and will be deployed soon, so merging this one.

**Pull request merged**